src/libNode/FinalBlockProcessing.cpp

```diff
@@ -282,6 +282,12 @@ void Node::InitiatePoW() {
   }
 
   SetState(POW_SUBMISSION);
+
```

Comment:
Should the new code be before SetState(POW_SUBMISSION); ?
Since we are disabling POW_SUBMISSION, should we keep in previous state or it's required to move to POW_SUBMISSION state?

Reply:
It ended up not mattering; I tried both before and after. As long as a node skips mining it basically stops progressing there.